pkg/cmd/root.go

```diff
@@ -21,11 +21,12 @@ var rootCmd = &cobra.Command{
 	SilenceUsage:  true,
 	SilenceErrors: true,
 	Annotations: map[string]string{
-		"get":     "api",
-		"post":    "api",
-		"delete":  "api",
-		"trigger": "webhooks",
-		"listen":  "webhooks",
+		"get":       "api",
+		"post":      "api",
+		"delete":    "api",
+		"trigger":   "webhooks",
+		"listen":    "webhooks",
+		"logs tail": "logs",
```

Comment:
I think this needs a template change as well. Does the help command print correctly?

Reply:
Good catch -- it does not. Will fix this.